giant_news/models/article.py

```diff
@@ -16,49 +16,40 @@
 __all__ = ["Tag", "Author", "Category", "Article", "ArticleQuerySet", "AbstractArticle"]
 
 
-class Tag(TimestampMixin):
-    """
-    Model to store a tag for the Article model
-    """
+class NameAndSlugAbstract(TimestampMixin):
```

Comment:
Assuming we don't want this to be included in the `__all__` above?

Reply:
added flake8 along with it